src/components/ButtonForm/index.js

```diff
@@ -4,16 +4,20 @@ import clsx from 'clsx';
 
 import styles from './styles.module.scss';
 
-function ButtonForm({ type, isFilled, children }) {
+function ButtonForm({ type, isFilled, isDivider, children }) {
   /* eslint-disable react/button-has-type */
   return (
-    <button type={type} className={clsx(styles.button, { [styles.buttonFill]: isFilled })}>
+    <button
+      type={type}
+      className={clsx(styles.button, { [styles.buttonFill]: isFilled }, { [styles.isDivider]: isDivider })}
```

Comment:
for now its ok, but you have to think about responsibilities in the future, this divider is not a common case nor a button responsibility